psd-web/spec/requests/user_completes_registration_spec.rb

```diff
@@ -101,19 +113,23 @@
         expect(user).not_to be_account_activated
       end
 
-      it "redirects to the root path" do
-        expect(response).to redirect_to(root_path)
+      it "does not set the mobile number as verified" do
+        expect(user.mobile_number_verified).to be false
```

Comment:
Suggestion (showing the possibility to be considered more than requesting any change):
```suggestion
        expect(user).not_to be_mobile_number_verified
```